velox/functions/prestosql/registration/GeometryFunctionsRegistration.cpp

```diff
@@ -55,24 +69,19 @@ void registerOverlayOperations(const std::string& prefix) {
       {{prefix + "ST_Union"}});
 }
 
+void registerAccessors(const std::string& prefix) {
+  registerFunction<StXFunction, double, Geometry>({{prefix + "ST_X"}});
```

Comment:
can you also add the new functions to documentation at: velox/docs/functions/presto/geospatial.rst